libs/utils/platforms/juno_energy.py

```diff
@@ -0,0 +1,109 @@
+from energy_model import ActiveState, EnergyModelNode, PowerDomain, EnergyModel
+
```

Comment:
[The newline is supposed to be there but the imports are in the wrong order](https://www.python.org/dev/peps/pep-0008/#imports), will rearrange